website/src/pages/examples/markdownPageExample.md

```diff
@@ -32,3 +34,19 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 <Tabs defaultValue="apple" values={[ {label: 'Apple', value: 'apple'}, {label: 'Orange', value: 'orange'}, {label: 'Banana', value: 'banana'} ]}><TabItem value="apple">This is an apple 🍎</TabItem><TabItem value="orange">This is an orange 🍊</TabItem><TabItem value="banana">This is a banana 🍌</TabItem></Tabs>
+
+## Import Mdx and Md files
+
+```js
+// *.md file
+import Comp from './_ignore.md';
+
+// *.mdx file
+import OtherComp from './_ignore_mdx.mdx';
+```
```

Comment:
That's cool to dogfood this feature on our example page. 

I'd rather have better, more realistic name/content than `_ignore.md`, current names could be confusing imho

Reply:
+ you should also add usage in the code block

I'd suggest something like:

## Import Mdx and Md files

```js
import Chapter1 from './_chapter1.md';

<Chapter1/>

import Chapter2 from './_chapter2.mdx';

<Chapter2/>
```

and document better this feature that's not widely known about MDX, as people won't it about it naturally